src/view/domconverter.js

```diff
@@ -266,7 +286,9 @@ export default class DomConverter {
 			if ( isInlineFiller( domNode ) ) {
 				return null;
 			} else {
-				return new ViewText( getDataWithoutFiller( domNode ) );
+				const textData = this._processDataFromDomText( domNode );
+
+				return textData === '' ? null : new ViewText( textData );
```

Comment:
Unrelated improvement. `domToView` is used in `domChildrenToView`, I just didn't want empty `ViewText` in view. However, the docs (`@returns` sentence) should be updated. Of course the drawback now is that if you want to just convert a dom Text instance and it is empty, you'll get `null` instance empty `ViewText`.